NoteFind/Views/OnlineGridCell.m

```diff
@@ -0,0 +1,30 @@
+//
+//  OnlineGridCell.m
+//  NoteFind
+//
+//  Created by Khloe Wright on 8/4/22.
+//
+
+#import "AppDelegate.h"
+#import "ErrorAlerts.h"
+#import "OnlineGridCell.h"
+
+@implementation OnlineGridCell
+
+-(void)awakeFromNib {
+    [super awakeFromNib];
+    UITapGestureRecognizer *tapRecognizer = [[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(doubleTappedCell:)];
+    tapRecognizer.delegate = self;
+    tapRecognizer.numberOfTapsRequired = 2;
+    tapRecognizer.numberOfTouchesRequired = 1;
+    [self addGestureRecognizer:tapRecognizer];
+}
+
+
+- (void)doubleTappedCell:(UITapGestureRecognizer *)tapRecognizer {
+    AppDelegate *appDelegate = (AppDelegate *)[UIApplication sharedApplication].delegate;
+    UIViewController *vc = [appDelegate getCurrentVC];
+    [ErrorAlerts confirmingDownload:vc withNote:_gridNote];
```

Comment:
Is there a reason the Error alert helpers can't get the current VC now if its globally accessible from the AppDelegate?

Reply:
i could definitely do this in the alert helper, i will make sure to change